eng/Versions.props

```diff
@@ -76,4 +76,8 @@
       <FileVersion>$(VersionPrefix).$(FileVersion.Split('.')[3])</FileVersion>
     </PropertyGroup>
   </Target>
+   <!-- SolutionPersistence -->
+  <PropertyGroup>
+	  <MicrosoftVisualStudioSolutionPersistenceVersion>1.0.4</MicrosoftVisualStudioSolutionPersistenceVersion>
```

Comment:
Do we need to self-service push to dotnet-public feed? https://dev.azure.com/dnceng/internal/_wiki/wikis/DNCEng%20Services%20Wiki/882/Updating-the-dotnet-public-feed